packages/terra-search-field/CHANGELOG.md

```diff
@@ -2,20 +2,23 @@
 
 ## Unreleased
 
+* Changed
+  * Fixed mobile keyboard not closing after pressing enter key on mobile devices.
+
 ## 3.74.5 - (May 3, 2022)
 
 * Changed
-  * Minor dependency version bump
+  * Minor dependency version bump.
```

Comment:
The `Minor dependency version bump` log is generated by this [script](https://github.com/cerner/terra-toolkit/blob/%40cerner/terra-open-source-scripts%401.12.0/packages/terra-open-source-scripts/src/terra-cli/prepare-for-release/updateChangelogForPackage.js#L26). If we prefer to add a period, we should make the change in this script instead of manually here to avoid inconsistencies with all other changelogs.

Reply:
Undid the changes since it's probably better as a separate conversation & PR and just adds noise here:
f8e54a4ab93abd094046383931e0a3c9c0a4dff7